src/store/getters/resource-getters.ts

```diff
@@ -24,7 +15,14 @@ export const getCompanyDisplayName = (state: StateIF): string => {
 
 /** The People and Roles object. */
 export const getPeopleAndRolesResource = (state: StateIF): any => {
-  return state.resourceModel.peopleAndRoles
+  const peopleAndRoles = state.resourceModel.peopleAndRoles
+  if (isFullRestorationFiling(state)) {
+    peopleAndRoles.rolesSubtitle = peopleAndRoles.rolesSubtitle[0]
+  }
+  if (isLimitedRestorationFiling(state)) {
+    peopleAndRoles.rolesSubtitle = peopleAndRoles.rolesSubtitle[1]
+  }
+  return peopleAndRoles
```

Comment:
Yes, this isn't super-general (filing-agnostic), but at least it keeps the components clean.

Ditto line 65 below.